pkg/tkn/tkn.go

```diff
@@ -147,3 +148,21 @@ func StartPipeline(pipelineName string, params map[string]string, workspaces map
 	log.Printf("Pipelinerun %s started", pipelineRunName)
 	return pipelineRunName
 }
+
+var Tkn_version_map map[string]string
+
+func UpdateVersionsMap() {
+	out, err := exec.Command("tkn", "version").Output()
```

Comment:
We do not use the `exec` directly. Please use the the method `MustSucceed`. You can refer to the function `StartPipeline` which is present in same file